FordLabs/PeopleMover
# Feature branch deploy

## Issue
Connects hot deploy feature branch (no story number as it was a note)

## What was done
- [x] Added gradle task to hot deploy front end and back end pcf apps

## How to test
1. Go through Mike's Setup file in sharepoint
2. *NOT ON VPN* Run ./gradlew hotDeployBranch at root of project (this will builid the app off of vpn)
3. *ON VPN* Run ./gradlew hotDeployBranch at root of project (sends off the apps to pcf)
4. Check the FeatureBranchUI app on pcf and access it to see your hot deployed branch!
